src/pages/available-equipment/Table/InventoryTable.jsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import LendBtn from '../../../components/btn/LendBtn';
+import {
+  TableContainer,
+  TableRow,
+  TableHeader,
+  TableColumn,
+  Divider,
+  ItemLabel,
+  ItemCount,
+  AvailableCount,
+  UnavailableCount,
+  TotalCount,
+} from './InventoryTable.styles';
+
+const InventoryTable = () => {
+  const [items, setItems] = React.useState([
```

Comment:
```suggestion
  const [items, setItems] = useState([
```

useState를 import 해서 사용하는게 좋아보입니다.

Reply:
그리고 items는 해당 데이터가 어떤 데이터인지 제대로 표현해주지 않습니다.
좀더 명확한 이름을 사용해주세요. 우리는 Equipments를 사용하고 있으니 이것도 좋습니다.